.github/workflows/python-package.yml

```diff
@@ -16,6 +16,44 @@ on:
   workflow_dispatch:
 
 jobs:
+  aim-webui-dist:
+    if: ${{ github.event_name == 'schedule' || github.actor == 'gorarakelyan' || github.actor == 'alberttorosyan' || github.actor == 'roubkar' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Docker & pull images
+        run: |
+          apt update && apt install -y docker.io
+          sudo systemctl enable --now docker
+          docker pull quay.io/pypa/manylinux_2_24_x86_64
+
+      - name: Use node v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Build UI
+        working-directory: ./aim/web/ui
+        run: |
+          npm i
+          CI=false npm run build
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+
+      - name: Build sdist
+        if: matrix.python-version == '3.9'
+        run: |
+          python aim/web/ui/setup.py sdist
+
+      - name: Publish wheel
+        env:
+          PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
I think we'll need a separate PyPI token for aim-webui package. I will setup one.

Reply:
please disregard this one